### DISCUSSION TITLE
onDateClick without contextual menu

### DISCUSSION THREAD

**Discussion by pasteko** [GuavaCZ/calendar#51] [Q&A]
Hello,
is it possible to have an example or add to the demo how to implement this?
https://github.com/GuavaCZ/calendar?tab=readme-ov-file#date-click-event

**Comment by pasteko** [comment]
I have to replace Fullcalendar, is it possible to just click on a date (without context menu) in the monthly to display the modal to create an event?
If yes how can I do that?

**Comment by lukas-frey** [comment]
Yeah, just don't use the context menu and in the onDateClick method mount the action you want, something along the lines of:

```
$this->mountAction('edit')
```
